lib/stubs/object_ml.c

```diff
@@ -0,0 +1,135 @@
+#include "lablqml.h"
```

Comment:
OCamlObject class takes care of any extra steps that might be needed for interaction between Qt and OCaml. Future proofs code using these bindings.

I must have made it from scratch, thus marked as new. There was a wild experimentation phase.